api/cpp/esp-idf/slint/src/slint-esp.cpp

```diff
@@ -18,15 +18,36 @@ static const char *TAG = "slint_platform";
 
 using RepaintBufferType = slint::platform::SoftwareRenderer::RepaintBufferType;
 
+template<typename PixelType>
```

Comment:
why do EspWindowAdapter need to be templated?

Reply:
Oops, good catch - thanks!